src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js

```diff
@@ -0,0 +1,362 @@
+    browser.assert.not.elementPresent('//*[contains(text(),"TestCom")]');
+    localGroupPopup.close();
+
+    // Remove All
+
+    clientLocalGroups.openBacDetails();
+    browser.waitForElementVisible(localGroupPopup);
+    browser.assert.elementPresent('//*[contains(text(),"TestGov")]');
+    browser.assert.elementPresent('//*[contains(text(),"TestOrg")]');
+
+    localGroupPopup.clickRemoveAll();
+    localGroupPopup.cancelMemberRemove();
+    browser.waitForElementNotVisible('//*[contains(@data-test, "dialog-title") and contains(text(), "Remove all members?")]');
+    browser.assert.elementPresent('//*[contains(text(),"TestGov")]');
+    browser.assert.elementPresent('//*[contains(text(),"TestOrg")]');
+
+
+    localGroupPopup.clickRemoveAll();
+    localGroupPopup.confirmMemberRemove();
+    browser.waitForElementNotVisible('//*[contains(@data-test, "dialog-title") and contains(text(), "Remove all members?")]');
+    browser.assert.not.elementPresent('//*[contains(text(),"TestGov")]');
+    browser.assert.not.elementPresent('//*[contains(text(),"TestOrg")]');
+
+    browser.end();
+
+  },
+  'Security server edit local group': browser => {
+    const frontPage = browser.page.ssFrontPage();
+    const mainPage = browser.page.ssMainPage();
+    const clientsTab = mainPage.section.clientsTab;
+    const clientInfo = mainPage.section.clientInfo;
+    const clientLocalGroups = clientInfo.section.localGroups;
+    const localGroupPopup = mainPage.section.localGroupPopup;
+    // Open SUT and check that page is loaded
+    frontPage.navigate();
+    browser.waitForElementVisible('//*[@id="app"]');
+
+    // Enter valid credentials
+    frontPage
+      .clearUsername()
+      .clearPassword()
+      .enterUsername(browser.globals.login_usr)
+      .enterPassword(browser.globals.login_pwd)
+      .signin();
+
+    // Open TestGov Internal Servers and local group details view
+    mainPage.openClientsTab();
+    browser.waitForElementVisible(clientsTab);
+    clientsTab.openTestService();
+    browser.waitForElementVisible(clientInfo);
+    clientInfo.openLocalGroupsTab();
+    browser.waitForElementVisible(clientLocalGroups);
+    clientLocalGroups.openCbbDetails();
+    browser.waitForElementVisible(localGroupPopup);
+
+    // Change description
+    localGroupPopup.changeDescription('');
+    localGroupPopup.clickHeader();
+    browser.assert.containsText('//div[contains(@class, "v-snack__content")]', 'Request failed with status code 500');
+    mainPage.closeSnackbar();
+    localGroupPopup.close();
+    browser.waitForElementVisible('//table[contains(@class, "details-certificates")]//tr[.//*[contains(text(),"cbb")] and .//*[contains(text(), "Group4")]]')
+    clientLocalGroups.openCbbDetails();
+    browser.waitForElementVisible(localGroupPopup);
+    localGroupPopup.changeDescription(browser.globals.test_string_300.slice(0,256));
```

Comment:
Would maybe be beneficial to test corner case: 255 characters long description _can_ be saved. Even though acceptance criteria does not require it.

Reply:
Implemented